common/src/io.cpp

```diff
@@ -215,112 +215,29 @@ pcl::concatenateFields (const pcl::PCLPointCloud2 &cloud1,
 }
 
 //////////////////////////////////////////////////////////////////////////
+template <typename PointT>
```

Comment:
Almost missed this one :sweat_drops: 

Template definition should be available in the header file.